cli/lib/kontena/cli/stacks/upgrade_command.rb

```diff
@@ -148,6 +148,11 @@ def execute
 
     def update_stack(name, data)
       return true if dry_run?
+      data.delete('errors')
+      data.delete('notifications')
+      data['services'].each do |svc|
+        svc['env'] = svc.delete('environment')
```

Comment:
I don't get this D:

Why use the wrong name in the `ServiceGenerator#parse_data`, and then mangle it in each command?

There's plenty of other things that are named subtly differently between the server API `data` and the stack YAML `options`.

Reply:
Because 'environment' is everywhere.

Reply:
I don't see anything in the actual CLI code that touches the generated service `'env'`? The `Kontena::Cli::Stacks::YAML::Reader` specs are easy to fix.

I think this should be reverted back to `env` in the `ServiceGenerator`, unless there's some other reason to change it...